filesmanager/settings/common.py

```diff
@@ -9,8 +9,9 @@
 ]
 
 
-def plugin_settings(settings):  # pylint: disable=unused-argument
+def plugin_settings(settings):
```

Comment:
have you tested this ?  I mean i'm not sure if this works with xblocks

Reply:
Yes, we have already used it in other XBlocks and it works correctly.